src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -65,6 +65,126 @@ public bool GetBooleanValue()
             }
         }
 
+        /// <summary>
+        /// Reads the next JSON token value from the source and parses it to an <see cref="int"/>.
+        /// Returns the value if the entire UTF-8 encoded token value can be successfully parsed to an <see cref="int"/>
+        /// value.
+        /// Throws exceptions otherwise.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        /// <exception cref="FormatException">
+        /// Thrown if the value of JSON token value is either of incorrect format or, is too large or too small for an <see cref="int"/>.
```

Comment:
`... incorrect format (for example if it contains a decimal or is written in scientific notation)...`